lms/envs/common.py

```diff
@@ -2274,6 +2274,9 @@ def _make_locale_paths(settings):
 
     # edx-drf-extensions
     'csrf.apps.CsrfAppConfig',  # Enables frontend apps to retrieve CSRF tokens.
+
+    # Program Enrollments
+    'lms.djangoapps.program_enrollments',
```

Comment:
Oooh, I think we may want to install this as a plugin, see e.g. https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/apps.py

I can find docs and help you wire it up correctly.